medical_insurance/views/medical_insurance_plan_view.xml

```diff
@@ -31,14 +34,16 @@
                         <notebook>
                             <page string="Plan Information">
                                 <group name="plan_info">
-                                    <field name="plan_number" />
-                                    <field name="product_id" />
-                                    <field name="insurance_company_id" />
-                                    <field name="insurance_affiliation" />
+                                    <field name="insurance_template_id" />
+                                    <field name="plan_number"/>
+                                    <field name="product_id"/>
+                                    <field name="insurance_company_id"/>
+                                    <field name="insurance_affiliation"/>
                                 </group>
                             </page>
                             <page string="Additional">
-                                <field name="notes" placeholder="Notes..." />
+                                <field name="notes" readonly="1"
```

Comment:
ups, because before I was testing the previous structure and trying to separate the access from the template. I'll delete that.